### DISCUSSION TITLE
Bora começar o sistema de achievements do aplicativo

### DISCUSSION THREAD

**Discussion by 404jv** [404jv/dev-landia#87] [Q&A] [answered]
Esse parte da Devlândia vai demorar um pouco pra sair do forno. Por isso, acho que a gente tem que está na mesma página antes de começar a por a mão na massa.

Primeiro, esse sistema vai mudar tudo que pensei sobre as tabelas feitas até então. Por meio desse [artigo](https://gamedevelopment.tutsplus.com/tutorials/how-to-code-unlockable-achievements-for-your-game-a-simple-approach--gamedev-6012) vou desenvolver a lógica do sistema. 

Se quiserem ler, e se tiver uma dúvida abra uma nova discussão aqui no repo. Me diz se faz sentido comentários.

**Comment by jvolima** [comment, marked as answer]
faz sentido sim, vamos focar nessa parte agora
